app/Http/Controllers/EntityController.php

```diff
@@ -60,9 +63,13 @@ public function store(CreateEntityRequest $request)
      * @param  \App\Models\Entity  $entity
      * @return \Illuminate\View\View
      */
-    public function show(Entity $entity)
+    public function show(Entity $entity): View
     {
-        $entity->load('pastProjects', 'currentProjects');
+        if (Route::currentRouteName() === \locale() . '.entities.show-projects') {
+            $entity->load('currentProjects');
+        } elseif (Route::currentRouteName() === \locale() . '.entities.show-projects') {
+            $entity->load('pastProjects', 'currentProjects');
```

Comment:
The two conditionals appear to be the same. 

Also I wonder if you could write this  as `request()->isRoute('entities.show-projects')`. You've done something similar in the blade templates.

See: 

- https://laravel.com/docs/8.x/requests#inspecting-the-request-path
- https://laravel.com/api/8.x/Illuminate/Http/Request.html#method_routeIs

Reply:
Good catch on the conditionals. Using `$request->isRoute()` means the `$request` variable would need to be set to a `Request` class instance representing the current request, which it isn't in that controller. I decided to use the Route facade instead. If you think there's a compelling reason to do so I could change it but it seemed like accessing the current request just to get the route name was more complex that checking it using the Route facade.

Reply:
When would these conditionals be hit?